maxprocs/maxprocs.go

```diff
@@ -95,7 +95,7 @@ func Set(opts ...Option) (func(), error) {
 
 	// Honor the GOMAXPROCS environment variable if present. Otherwise, amend
 	// `runtime.GOMAXPROCS()` with the current process' CPU quota if the OS is
-	// Linux, and guarantee a minimum value of 2 to ensure efficiency.
+	// Linux, and guarantee a minimum value minGOMAXPROCS.
```

Comment:
Please update to mention the default value for this ( which is `1` not `2` ). Also since `minGOMAXPROCS` is unexported, this should instead mention the exported API  for changing it... which I'd have to go digging through the code to remember what it is right now, hence why this doc string should  mention it ;-)